linux_os/guide/services/nfs_and_rpc/nfs_configuring_clients/mounting_remote_filesystems/mount_option_krb_sec_remote_filesystems/ansible/shared.yml

```diff
@@ -5,13 +5,21 @@
 # disruption = medium
 
 - name: "Get nfs and nfs4 mount points, that don't have Kerberos security option"
-  shell: grep -E "[[:space:]]nfs[4]?[[:space:]]" /etc/fstab | grep -v "sec=krb5:krb5i:krb5p" | awk '{print $2}'
+  shell: |
+    set -o pipefail
+    grep -E "[[:space:]]nfs[4]?[[:space:]]" /etc/fstab | grep -v "sec=krb5:krb5i:krb5p" | awk '{print $2}'
+  args:
+    warn: False
   register: points_register
   check_mode: no
   changed_when: False
 
 - name: "Add Kerberos security to mount points"
-  shell: awk '$2=="{{ item }}"{$4=$4",sec=krb5:krb5i:krb5p"}1' /etc/fstab > fstab.tmp && mv fstab.tmp /etc/fstab
+  shell: |
+    set -o pipefail
+    awk '$2=="{{ item }}"{$4=$4",sec=krb5:krb5i:krb5p"}1' /etc/fstab > fstab.tmp && mv fstab.tmp /etc/fstab
+  args:
+    warn: False
```

Comment:
```suggestion
    executable: /bin/bash
    warn: False
```

Reply:
undid this change as there is no pipe `|` in the shell command.